source/cluster_manager/matcher.js

```diff
@@ -97,6 +127,25 @@ var videoMatcher = function () {
             }
             return (encodeOk && decodeOk);
         });
+
+        for (var i in availableCandidates) {
```

Comment:
Is this part same as isp & region selection in `webrtcMatcher`.
I suggest wrap the same logic in a function and call it.
Something like:
`var ispRegionMatcher = new webrtcMatcher();
 var candidates = ispRegionMatcher.match(preference, workers, candidates);`